stand/libsa/ufs.c

```diff
@@ -390,10 +382,7 @@ buf_write_file(f, buf_p, size_p)
  * the location in the buffer and the amount in the buffer.
  */
 static int
-buf_read_file(f, buf_p, size_p)
-	struct open_file *f;
-	char **buf_p;		/* out */
-	size_t *size_p;		/* out */
+buf_read_file(struct open_file *f, char **buf_p, size_t *size_p)		/* out */
```

Comment:
Same as my comment for block_map above (though maybe this one isn't > 80 characters.